src/bundle/Controller/ContentTypeController.php

```diff
@@ -385,6 +386,41 @@ public function editAction(Request $request, ContentTypeGroup $group, ContentTyp
         ]);
     }
 
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup $group
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function copyAction(ContentTypeGroup $group, ContentType $contentType): Response
```

Comment:
Hmmm I'm missing here CSRF protection....

Reply:
@adamwojs do you have an example of handling the CSRF in the ezplatform-admin-ui?

Reply:
I can create a simple form in controller to create and validate a CSRF.

Reply:
Yup, that's what we do too. You could also drop `GET` action in routing then.